expression/bench_test.go

```diff
@@ -511,13 +512,27 @@ func genVecBuiltinFuncBenchCase(ctx sessionctx.Context, funcName string, testCas
 	return baseFunc, input, result
 }
 
-// testVectorizedBuiltinFunc is used to verify that the special vectorized
+// testVectorizedBuiltinFunc is used to verify that the vectorized
 // expression is evaluated correctly
 func testVectorizedBuiltinFunc(c *C, vecExprCases vecExprBenchCases) {
+	// If you just want to run a specified vectorized built-in function.
+	// You should input "export vec_test_func_sig=XXXsig" first, which "XXXsig" means
+	// the expression function signature you want to test.
+	// If you want to test all of them, just ignore it or "export vec_test_func_sig="
+	// The following benchmark is also like this.
+	TestFuncName := os.Getenv("vec_test_func_sig")
```

Comment:
Just wondering can we use golang flag? It can be passed with `go test`.

Reply:
I think your opinion is better, I will do some investigation

Reply:
update. PTAL